lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -86,8 +86,14 @@ def index(self, trans, **kwds):
     @expose_api_anonymous_and_sessionless
     def show(self, trans, id, **kwd):
         """
-        GET /api/tools/{tool_id}
-        Returns tool information, including parameters and inputs.
+        GET /api/tools/{tool_id}/show
```

Comment:
Sorry, I'm stupid, it was actually correct before

Reply:
Okay ^^", I do that